### ISSUE TITLE
ASceneFrameCallback onPreDraw not called

### ISSUE THREAD

**Issue by TBog** [Rajawali/Rajawali#1840]
#### Rajawali Version or Branch
1.0.186-snapshot
#### Device and Android Version
device SM-T705 android 5.0.2

#### Summary
method onPreDraw from ASceneFrameCallback not called

#### Steps to Reproduce
    ASceneFrameCallback mCB = new ASceneFrameCallback()
    {
        @Override
        public void onPreFrame( final long sceneTime, final double deltaTime )
        {
        }

        @Override
        public void onPreDraw( final long sceneTime, final double deltaTime )
        {
            Log.d( "CB", "onPreDraw" );
        }

        @Override
        public void onPostFrame( final long sceneTime, final double deltaTime )
        {
        }

        @Override
        public boolean callPreDraw()
        {
            return true;
        }
    };
mRenderer.getCurrentScene().registerFrameCallback( mCB );
Will not see the debug trace unless I also override callPreFrame to return true

**Comment by contriteobserver** [comment]
This is expected behaviour, return true to `callPreFrame()`, `callPreDraw()` or `callPostFrame()` to add a scene callback, as per `Scene.registerFrameCallback()`
```
                if (callback.callPreFrame()) mPreCallbacks.add(callback);
                if (callback.callPreDraw()) mPreDrawCallbacks.add(callback);
                if (callback.callPostFrame()) mPostCallbacks.add(callback);
```
Please clarify why this behaviour is an issue.

**Comment by jwoolston** [comment]
@contriteobserver is 100% correct. This is expected and documented.

**Comment by TBog** [comment]
The Log.d is in onPreDraw and callPreDraw returns true. Also I have registered the callback.
The issue is in the Scene code of the Rajawali version 1.0.186-snapshot

**Comment by rpicolet** [comment]
Yep, that was fixed by #1601

**Comment by jwoolston** [comment]
Ah, @rpicolet is referring to a copy/paste error.
